assembly/package-res/Kitchen.bat

```diff
@@ -1,7 +1,8 @@
 @echo off
+SET initialDir=%cd%
 setlocal
 pushd %~dp0
 SET STARTTITLE="Kitchen"
 SET SPOON_CONSOLE=1
-call Spoon.bat -main org.pentaho.di.kitchen.Kitchen %*
+call Spoon.bat -main org.pentaho.di.kitchen.Kitchen -initialDir %initialDir%\ %*
```

Comment:
This will fail if the folder has spaces in it. I know from past experience. It needs to be quoted